diff --git a/web/src/lib/components/asset-viewer/detail-panel.svelte b/web/src/lib/components/asset-viewer/detail-panel.svelte
index 75f9e75..b160e26 100644
--- a/web/src/lib/components/asset-viewer/detail-panel.svelte
+++ b/web/src/lib/components/asset-viewer/detail-panel.svelte
@@ -499,10 +499,10 @@
           <div class="flex flex-col items-center gap-1">
             <p class="font-bold">{lat.toPrecision(6)}, {lon.toPrecision(6)}</p>
             <a
-              href="https://www.openstreetmap.org/?mlat={lat}&mlon={lon}&zoom=13#map=15/{lat}/{lon}"
+              href="https://www.google.com/maps/search/?api=1&query={lat},{lon}"
               target="_blank"
               class="font-medium text-primary underline focus:outline-none"
             >
-              {$t('open_in_openstreetmap')}
+              Google Maps
             </a>
           </div>
         {/snippet}
